pallets/usage/src/tests.rs

```diff
@@ -112,6 +112,27 @@ fn wait_list_contents() -> Vec<(StoredDispatch, u32)> {
     .collect()
 }
 
+fn decode_validate_usage_call(encoded: &[u8]) -> crate::Call<Test> {
+    let mut encoded = <&[u8]>::clone(&encoded);
+    let extrinsic: Extrinsic = Decode::decode(&mut encoded).unwrap();
+
+    let call = extrinsic.call;
+    let inner = match call {
```

Comment:
Why a separate variable?
```suggestion
    let inner = match extrinsic.call {
```

Reply:
ok